UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -397,6 +400,16 @@ public virtual void ToxinBreathinCheck(GasMix gasMix)
 			}
 		}
 
+		private void BreathInEffects(GasMix mix)
+		{
+			var reagents = mix.GetReagentMix();
+			foreach (var reaction in ReagentContainerBody.ReactionSet.reactions)
+			{
+				if (reaction.IsReactionValid(reagents) == false) continue;
+				reaction.ForceApply(reagents, LivingHealthMaster.gameObject);
```

Comment:
Sure you can just use the reagent Makes to do this?

Reply:
<img src="https://media4.giphy.com/media/GRk3GLfzduq1NtfGt5/giphy-downsized-medium.gif"/>

Reply:
actually wait no
no we cant

it needs a metabolism component, this skips the metabolism part and makes the lung responsible for checking and running the code for that

Reply:
why?

Reply:
> it needs a metabolism component